tox.ini

```diff
@@ -1,9 +1,7 @@
 [tox]
-envlist = py27, py3
+envlist = py36, py37, py38, py39, py310
```

Comment:
I think it's enough to only test against the lowest and the highest version we want to support.